great_expectations/datasource/fluent/data_connector/file_path_data_connector.py

```diff
@@ -212,7 +212,8 @@ def _get_unfiltered_batch_definition_list(
         batch_definition_set = set()
         # if the batch request hasn't specified a batching_regex, fallback to a default
         if batch_request.partitioner:
-            batching_regex = self._preprocess_batching_regex(batch_request.partitioner.regex)
+            # --- HEY JOSH! ---: Is this where we want to preprocess?
+            batching_regex = batch_request.partitioner.regex
```

Comment:
I've annotated every instance of `batching_regex` in this file (and removed all the preprocessing until we determine which ones are necessary). The issue we were seeing was that we'd accidentally preprocess too many times.

If we look at the usages of `batching_regex` within this file, it is evident that we may hit one, multiple, or none of these code paths based on usage. I don't think it is as simple as adding preprocessing to each instance.

I'll probably have a better idea after stepping away for a bit.

Reply:
resolved, thanks! 🥳